lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Transactions.hs

```diff
@@ -533,6 +549,178 @@ spec = do
                 (#balance . #available)
                 (`shouldBe` Quantity (faucetAmt - feeEstMax - amt)) ra2
 
+    it "TRANS_EXTERNAL_01 - Single Output Transaction" $ \ctx -> do
+        wFaucet <- fixtureWallet ctx
+        let amt = (10_000_000 :: Natural)
+
+        let walletPostData = Json [json| {
+                "name": "empty wallet",
+                "mnemonic_sentence": #{mnemonics15},
+                "passphrase": #{fixturePassphrase}
+                } |]
+        r1 <- request @ApiWallet ctx (Link.postWallet @'Shelley) Default walletPostData
+        verify r1
+            [ expectSuccess
+            , expectResponseCode HTTP.status201
+            , expectField
+                (#balance . #getApiT . #available) (`shouldBe` Quantity 0)
+            ]
+        let (_, Right wSrc) = r1
+
+        -- 1. mnemonic15 phrase: recovery-phrase-src.prv
+        -- --> network empty cause mean expire private finger accident session problem absurd banner stage void what
+        -- 2. corresponding root key:
+        --- $ cat recovery-phrase-src.prv | cardano-address key from-recovery-phrase Shelley > root-src.prv
+        -- --> xprv1dqjlgjd6n9n63guasjpp0q0p93v3lgwenwf2s8z6erzzqxgqv4qnsyk4mrh8z5nz3fesun538g3epe4l8j8rezfdvmns78sp796rsflp980qp6gqgh0uzyxtad0smxd88wt8a9djhkvq8zsvp2gzzycmpq927903
+        -- 3. staking private key:
+        --- $ cat root-src.prv | cardano-address key child 1852H/1815H/0H/2/0 > stake-src.prv
+        -- --> xprv1uz7ghz6969v3ns0veeddflaj9hr0q5dwnzgep7rznt72cvcqv4qa2l9kfqsrzsg72zl89wyty42jq4a6n6l2mnuj4zfv578jvh8tm8pa439z46e6qmhhqrwy40deer2dkcyr44u5ze4awfnhykgf5j94u59rcpdg
+        -- 4. delegation address to which we send 10 ada (index 1 and for network tag 1):
+        --- $ cat root-src.prv \
+        --- | cardano-address key child 1852H/1815H/0H/0/1 \
+        --- | cardano-address key public \
+        --- | cardano-address address payment --network-tag 1 \
+        --- | cardano-address address delegation $(cat stake-src.prv | cardano-address key public)
+        --- --> addr1q9wteuqmnkywcz9zefjyp0wn3tctz9xgxm8fpcmyzn9uypknudck0fzve4346yytz3wpwv9yhlxt7jwuc7ytwx2vfkyq75lt2a
+        payload1 <- mkTxPayload ctx wSrc amt fixturePassphrase
+
+        (_, ApiFee (Quantity feeMin) (Quantity feeMax)) <- unsafeRequest ctx
+            (Link.getTransactionFee @'Shelley wFaucet) payload1
+
+        r2 <- request @(ApiTransaction n) ctx
+            (Link.createTransaction @'Shelley wFaucet) Default payload1
+
+        verify r2
+            [ expectSuccess
+            , expectResponseCode HTTP.status202
+            , expectField (#amount . #getQuantity) $
+                between (feeMin + amt, feeMax + amt)
+            , expectField (#direction . #getApiT) (`shouldBe` Outgoing)
+            , expectField (#status . #getApiT) (`shouldBe` Pending)
+            ]
```

Comment:
What exactly is this for :thinking: ? How does it relate to the testing of external tx?